images/docker-compose.yml

```diff
@@ -26,7 +26,7 @@ services:
       POSTGRES_DB: core
 
   stellar-core-2:
-    image: kinecosystem/stellar-core:kinecosystem-v2.0.0-stellar-v9.2.0
+    image: kinecosystem/stellar-core:latest
```

Comment:
Same. Let's hardcode the versions in this file.

Reply:
blockchain team removes their docker images a lot so the tests fail. latest will always be there